tests/app/conftest.py

```diff
@@ -929,189 +921,9 @@ def template_folder_cleanup(
-
-    Note that Notification instances have foreign keys to TemplateHistory instances rather than
-    Template instances.
-    """
-
-    templates = []
-    template_data = sample_template_helper(
-        f'session e-mail template history {worker_id}', EMAIL_TYPE, sample_service(), sample_user()
-    )
-    template_history = TemplateHistory(**template_data)
-    notify_db.session.add(template_history)
-    notify_db.session.commit()
-    templates.append(template_history)
-
-    yield template_history
-
-    for template in templates:
-        notify_db.session.delete(template)
-        notify_db.session.commit()
-
-
-@pytest.fixture
-def sample_template_without_sms_permission(notify_db_session, sample_service, sample_template):
-    service = sample_service(service_permissions=[EMAIL_TYPE], check_if_service_exists=True)
-    template = sample_template(service=service, template_type=SMS_TYPE)
-    yield template
-
-    # Teardown
-    template_history = notify_db_session.session.get(TemplateHistory, (template.id, template.version))
-    notify_db_session.session.delete(template_history)
-
-    template_redacted = notify_db_session.session.get(TemplateRedacted, template.id)
-    notify_db_session.session.delete(template_redacted)
-    notify_db_session.session.delete(template)
-    notify_db_session.session.commit()
-
-
-@pytest.fixture
-def sample_template_with_placeholders(sample_template):
-    return sample_template(content='Hello (( Name))\nYour thing is due soon')
-
-
-@pytest.fixture
-def sample_sms_template_with_html(sample_service, sample_template):
-    # deliberate space and title case in placeholder
-    sample_service.prefix_sms = True
-    return sample_template(sample_service, content='Hello (( Name))\nHere is <em>some HTML</em> & entities')
-
-
-@pytest.fixture
-def sample_template_without_email_permission(sample_service, sample_template):
-    service = sample_service(service_permissions=[SMS_TYPE], check_if_service_exists=True)
-    return sample_template(service=service, template_type=EMAIL_TYPE)
-
-
-@pytest.fixture
-def other_sample_template(notify_db_session):
-    service = create_service(
-        service_permissions=[SMS_TYPE], service_name='OTHER_SMS_SERVICE', check_if_service_exists=False
-    )
-    return create_template(service, template_type=SMS_TYPE, template_name='OTHER_SMS_TEMLATE')
-
-
-@pytest.fixture
-def sample_letter_template(sample_service_full_permissions, sample_template):
-    return sample_template(service=sample_service_full_permissions, template_type=LETTER_TYPE, postage='second')
-
-
-@pytest.fixture
-def sample_trial_letter_template(sample_service_full_permissions, sample_template):
-    sample_service_full_permissions.restricted = True
-    return sample_template(service=sample_service_full_permissions, template_type=LETTER_TYPE)
+def sample_letter_template(sample_service, sample_template):
+    service = sample_service(service_permissions=[LETTER_TYPE], check_if_service_exists=True)
```

Comment:
"check_if_service_exists" is superfluous because there's no "service_name" value for which to check.
```suggestion
    service = sample_service(service_permissions=[LETTER_TYPE])
```